.circleci/config.yml

```diff
@@ -86,6 +86,20 @@ jobs:
           name: Trigger a release if the latest commit is a release commit
           command: |
             yarn shipjs trigger
+  "prepare release":
+    <<: *node_v12
+    steps:
+      - checkout
+      - run: *install_yarn_version
+      - restore_cache: *restore_yarn_cache
+      - run: *run_yarn_install
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Prepare release
+          command: |
+            git config --global user.email "eunjae.lee@algolia.com"
```

Comment:
We could put any name here right? If that's the case, I would advocate for no PII if possible, so something like `events-ingestion-team@algolia.com`.

Reply:
Yeah actually, the InstantSearch team has a shared email address for this purpose. 

something like `events-ingestion@algolia.com / Algolia`? Not sure if that email address is suitable for this process 🤔

Reply:
@JonathanMontane 

https://github.com/algolia/places/blob/38e8f30fd55cecdc723f096abd8c225e335f4e81/.travis.yml#L4-L5

what about that?

Reply:
probs okay 🙂